app/src/lib/git/diff.ts

```diff
@@ -83,7 +83,7 @@ const imageFileExtensions = new Set(['.png', '.jpg', '.jpeg', '.gif', '.ico'])
  * Render the difference between a file in the given commit and its parent
  *
  * @param commitish A commit SHA or some other identifier that ultimately dereferences
- *                  to a commit.
+ *                  a commit.
```

Comment:
I think the original is correct. The commitish dereferences **to** something. It does not dereference something.

Reply:
you're totally right @j-f1 thanks for pointing that out